gutenberg/_domain_model/exceptions.py

```diff
@@ -23,9 +23,5 @@ class CacheAlreadyExists(Error):
     pass
 
 
-class CacheNotRemovable(Error):
```

Comment:
Built-in python errors (almost) all end in Exception, Error, or Warning (https://docs.python.org/2/library/exceptions.html). See also PEP8: https://www.python.org/dev/peps/pep-0008/#exception-names